share/revng/test/tests/analysis/ImportFromCAnalysis/struct-with-a-rare-type/expected-error.txt

```diff
@@ -0,0 +1,7 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: Unsupported QualType.
```

Comment:
What's a `QualType`? I know it's `clang`'s stuff but this is not informative for users. Perhaps print the type name?

Reply:
That was a catch all error for failing to parse whatever clang spitted out. I went with:
```
import-from-c: Unsupported type.
import-from-c: This error indicates that the type system clang emitted contains something we either decided not to allow in the model OR do not support yet.
import-from-c failed: Unable to parse the type of struct field #0.
```

Reply:
Ok, but let's this less verbose and compact everything in a single line
```
import-from-c: unsupported type: C type provided for struct field #0 is not supported in the model
```

Reply:
Wait, don't forget that the real error
```
import-from-c failed: Unable to parse the type of struct field #0.
```

is emitted elsewhere, so it would be somewhat hard to get rid of it.

What about something like:
```
import-from-c: The type is cannot be represented in the model.
import-from-c failed: Unable to parse the type of struct field #0.
```
?

Reply:
> that the real error is emitted elsewhere

Ah right.

I like your proposal then. `is cannot ` is a typo though.
Besides that, I approve this

Reply:
Done (forgot to close).